wikimedia/wikipedia-ios
# Build server: use App Store Connect API key

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T271350

### Notes
* The key is on the server - and is not committed to the repo.

### Test Steps
1. Using jenkins, build a build - using this branch. Ensure it's uploaded to app store connect.
